Fix: reimplement sliding panel

matejglejtek opened the pull request:
Replace sliding panel implemented with the _sliding_panel_ package with new one using the [_sliding_sheet_](https://pub.dev/packages/sliding_sheet).

marianhlavac commented:
Please avoid using unverified abandoned packages. The repository of the package also seem to disappeared from GitHub.

matejglejtek commented:
The package I used originally was abandoned and is not compatible with Flutter 3.10, but it can be freely used and modified under MIT license. Because of this, I removed the package and used part of its modified implementation directly in the project. I also included original license in the project and in 3rd party licenses available from about dialog.
